torchreid/apis/export.py

```diff
@@ -50,7 +50,7 @@ def export_onnx(model, cfg, output_file_path='model', disable_dyn_axes=True,
         output_file_path += '.onnx'
 
     register_op("group_norm", group_norm_symbolic, "", opset)
-    register_op("hardsigmoid", hardsigmoid, "", opset)
+    register_op("hardsigmoid", hardsigmoid_symbolic, "", opset)
```

Comment:
Is it works now with the opset less than 14? Have you checked the accuracy after conversion to IR?

Reply:
It doesn't depend on the opset version and also barely can touch the accuracy, because MO simply matches ops by names and signatures of args. We don't need v14 to export models with hardsigmoid and can settle on v9 and pytorch 1.8.1.